resources/views/details-box-mobile.blade.php

```diff
@@ -1,8 +1,8 @@
 <div class="flex justify-between">
     <div class="{{ $titleClass ?? 'w-32' }}">{{ $title }}</div>
 
-    <div class="flex items-center space-x-3">
-        <div>{{ $slot }}</div>
+    <div class="flex items-center space-x-3 {{ $wrapperClass ?? false }}">
+        <div class="{{ $slotClass ?? 'false' }}">{{ $slot }}</div>
```

Comment:
won't this result in a `class="false"` entry?

Reply:
Oh my bad, that slipped.